### PR TITLE
Consolidated MigMigration and MigStage.

### PR DESCRIPTION
**Pull request by jortel**
Adds MigMigration.staging.
Deletes MigStage.
Deletes velerorunner package.

### CODE REVIEW

**Comment by jortel** [comment]
> Looks good to me. I had a minor question about terminology -- it looks like some things have shifted from "stage" to "staging" with the move here. Is that intentional?

Yes.  _stage_ seemed  more like a noun than an adjective.  That said, I don't feel strongly and a little on the fence myself.

**Comment by sseago** [comment]
Yeah, I'm fine either way too. Whatever will cause the least amount of confusion when looking at UI and API together probably makes the most sense.

**Comment by jortel** [comment]
After discussion, on forum-mig, it seems like consensus is `MigMigration.Stage` but make it _required_.
Pushed.

File: config/samples/mig-migration.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
   name: migmigration-sample
   namespace: mig
 spec:
-
+  staging: false
```
Comment (dymurray) [inline config/samples/mig-migration.yaml]: This needs to be changed to `stage`

Comment (jortel) [reply config/samples/mig-migration.yaml]: Fixed.

**Pull request merged**